internal/batches/executor/coordinator.go

```diff
@@ -72,6 +72,9 @@ func NewCoordinator(opts NewCoordinatorOpts) *Coordinator {
 		Parallelism: opts.Parallelism,
 		Timeout:     opts.Timeout,
 		TempDir:     opts.TempDir,
+		WriteToCache: func(ctx context.Context, stepResult execution.AfterStepResult, task *Task) error {
```

Comment:
```suggestion
		WriteStepCacheResult: func(ctx context.Context, stepResult execution.AfterStepResult, task *Task) error {
```

to align with the comment below?

Reply:
Sounds good. I'll commit this suggestion.